sig-docs/annual-activities.md

```diff
@@ -0,0 +1,84 @@
+are willing to continue contributing.
+
+This can be done by using the [maintainers tool](https://github.com/kubernetes-sigs/maintainers) which helps parse the file for 
+activity levels of those listed from GitHub and [devstats](https://k8s.devstats.cncf.io).
+
+### Steps
+
+- Set up the [maintainers tool](https://github.com/kubernetes-sigs/maintainers) along with its prerequisites locally by following the instructions.
+- Clone the [k/website](https://github.com/kubernetes/website) repository on your machine.
+- Create a new branch locally. This is required since there are nested OWNERS_ALIASES files within the repository.
+- Switch to the newly created branch and delete all nested OWNERS_ALIASES files. Retain the OWNERS_ALIASES file at the top of the repository root only.
+Without this, the maintainers tool will not produce the required output.
+- Run the maintainers tool against this new copy of the website repo locally.
+  - Tip: While executing the command, ensure you exclude the following folks by using the `--exclude` flag:
+      - Security Response Committee members,
+      - Release engineering approvers and reviewers,
+      - Co-chairs and tech leads of SIG Docs, SIG Release, and SIG Security
+- You'll get a list of SIG Docs approvers/reviewers with no contributions and with low reviews/approvals.
+- Submit a separate PR each for the list with no contributions and with low reviews/approvals and seek feedback. The timeframe for this activity should be ~2 weeks per PR.
+  - Tip: The lists will likely be long. It'd be easier if you raise the second one after the first PR is merged.
+  - Example PRs:
+    - https://github.com/kubernetes/website/pull/38719
+    - https://github.com/kubernetes/website/pull/38853
+- Socialize the activity within the [SIG Docs google group](https://groups.google.com/u/1/g/kubernetes-sig-docs) and the [#sig-docs slack channel](https://kubernetes.slack.com/messages/sig-docs). Add it as an item to the upcoming meeting agenda.
+- Work with all stakeholders to get the PRs merged, ideally before the end of the year.
+- In case of issues with merging due to conflicts with localization initiatives, such as lack of minimum number of approvers/reviewers,
+seek consensus with the localization subproject and wider SIG Docs community in the next year.
+  
+## Create new PR Wrangler schedule
+
+Preferably, before the second week of December, [the PR Wrangler schedule](https://github.com/kubernetes/website/wiki/PR-Wranglers) for the upcoming year needs to be released. 
+
+This needs to be done along with or after the pruning of the OWNERS_ALIASES file detailed above.
+
+SIG co-chairs, tech leads, and localization subproject leads have access to edit the [k/website wiki](https://github.com/kubernetes/website/wiki/).
+
+### Steps:
+
+- Create a new page under [PR Wranglers](https://github.com/kubernetes/website/wiki/PR-Wranglers) for the upcoming year.
```

Comment:
Should we also have a note about nominating new wranglers for the next yr? I'm not sure what that process looks like, but i noticed we have a "call for nominations" note for Issue wranglers.

Reply:
@a-mccarthy : Normally, we do not nominate PR Wranglers. The source for the PR Wrangling rota is the OWNERS_ALIASES file and the folks that are listed in the rota are approvers/reviewers for the English website.